mcodingbot/plugins/info.py

```diff
@@ -7,10 +7,15 @@
 plugin = Plugin()
 
 
+def nat_round(__x: float) -> int:
```

Comment:
```suggestion
def natural_round(__x: float) -> int:
```
could use a docstring on why its round. Something along the lines of. `"""rounds .5 up instead of down like round()"""`